asyncapi/document.go

```diff
@@ -0,0 +1,47 @@
+package asyncapi
```

Comment:
Btw, this is a candidate to be moved to https://github.com/asyncapi/parser-go at some point.